src/orion/core/worker/experiment.py

```diff
@@ -202,7 +202,7 @@ def reserve_trial(self, score_handle=None):
 
         return selected_trial
 
-    def push_completed_trial(self, trial):
+    def push_completed_trial(self, trial, status='completed'):
```

Comment:
```python
experiment.push_completed_trial(trial, 'reserved')
```
:thinking: 
If the behavior is generic, the name of the method should be generic too.

Reply:
I agree, but I am not courageous enough to break this kind of API. How about `finalize_trial` that I have suggested above?

Reply:
See my new comment above. :slightly_smiling_face: